ntoskrnl/mm/ARM3/i386/init.c

```diff
@@ -493,7 +493,7 @@ MiInitMachineDependent(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
 
     /* Allocate a page for hyperspace and create it */
     MI_SET_USAGE(MI_USAGE_PAGE_TABLE);
-    MI_SET_PROCESS2("Kernel");
+    MI_SET_PROCESS(0);
```

Comment:
```suggestion
    MI_SET_PROCESS(NULL);
```

OK there are many other places like that in the PR, that need the change (PEPROCESS is a pointer, not a number).